multiplatform-crypto-libsodium-bindings/src/nativeMain/kotlin/com/ionspin/kotlin/crypto/pwhash/PasswordHash.kt

```diff
@@ -108,7 +109,7 @@ actual object PasswordHash {
             password,
             password.length.convert()
         )
-        return result == 0
+        return result.isLibSodiumSuccessCode()
```

Comment:
Also caught by CI, a type here it should be 'isLibsodiumSuccessCode' but the sodium was capitalized